modules/formulize/include/functions.php

```diff
@@ -8051,3 +8073,50 @@ function generateSelfReferenceExclusionSQL($entry_id, $fid, $sourceFid, $ele_val
 	}
 	return $selfReferenceExclusion;
 }
+
+/**
+ * Record information in a log file. One log file per day. Log file location is a Formulize preference. Log file storage duration is a Formulize preference.
+ *
+ * @param array $data Key-Value pairs that should be written to the log entry
+ * @return int|boolean Returns the number of bytes written to the file, or false on failure
+ */
+function writeToFormulizeLog($data) {
+
+	// initialize the configuration settings
+	static $formulizeConfig = false;
+	static $formulizeLogFileLocation = XOOPS_ROOT_PATH.'/logs';
+	static $formulizeLogFileStorageDuration = 168;
```

Comment:
To avoid ambiguity I might suffix with the unit of measurement. 
```suggestion
	static $formulizeLogFileStorageDurationHours = 168;
```